src/client/pages/OfferNew/Checkout/SignDisclaimer.tsx

```diff
@@ -2,10 +2,11 @@ import React from 'react'
 import styled from '@emotion/styled'
 import ReactMarkdown from 'react-markdown/with-html'
 import { colorsV3 } from '@hedviginsurance/brand/dist'
-import { InsuranceTermType } from 'data/graphql'
 import { useTextKeys } from 'utils/textKeys'
 import { OfferData } from 'pages/OfferNew/types'
 import { isNorwegian, isSwedish } from 'pages/OfferNew/utils'
+import { useCurrentLocale } from 'components/utils/CurrentLocale'
+import { InsuranceTermType } from '../../../data/graphql'
```

Comment:
Why can't VSCode get auto imporst right 😄
```suggestion
import { InsuranceTermType } from 'data/graphql'
```

Reply:
Actually I had to manually change it since it wouldn't compile with `'data/graphql'` 🤷‍♀️

Reply:
And since there are so many issues with this I was thinking maybe we should just go for relative paths always? Or is that a bad idea?

Reply:
hm that is weird 🤔   does it complain in any of all the other occurrences where we import from 'data/graphql'?

Reply:
I'd prefer to not have relative paths but the setup is not working so smooth 🙈

Reply:
No wait, maybe it's just me being confused - it should be `'src/client/data/graphql'` right? Which is actually what I think VSCode auto imports from but I thought it was wrong 😳

Reply:
Hm OK so we're using both `'src/client/data/graphql'` and `'data/graphql'` 🤔 
![image](https://user-images.githubusercontent.com/42962286/110915355-cfe04500-8317-11eb-9d63-2a6a3c582682.png)
![image](https://user-images.githubusercontent.com/42962286/110915399-e090bb00-8317-11eb-91e9-14b12e3fe761.png)

But now when I changed back to `'data/graphql'` it worked for some reason 🤷‍♀️

Reply:
computers are wierd 😼
![source (2)](https://user-images.githubusercontent.com/6661511/110920509-c5c14500-831d-11eb-9333-6b39fb2fc28d.gif)